README.md

```diff
@@ -1 +1,45 @@
-# leaderboard
\ No newline at end of file
+# Leaderboard
+> "Leaderboard" is an app that keeps a record of a leaderboard by making use of a site that provides an API for that purpose. It allows to add and remove scores from different browsers or locations and keep them safe.
+
+## Built With
+
+- HTML, CSS and JavaScript
+- Using GITHub, GitFlow, Visual Studio Code, Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these steps:
+
+### Setup
+
+- Clone the repository to your hard drive.
+- Use your browser of choice to see the results (./dist/index.html).
+- Execute npm install to be able to make changes.
+- You can run the live server by executing 'npm start'.
+- Use Visual Code to edit './src/index.html' and './src/index.js' files.
+- Use 'npm run build' to build the dist folder from the changed src files.
+### Deployment
+
+- Change your 'webpack.config.js' mode from development to production.
+- Execute 'npm run build'
+- Deploy '.dist/' contents to your http server.
+### Live Demo
+
+[https://acolombo1.github.io/leaderboard/dist/](https://acolombo1.github.io/leaderboard/dist/)
+## Author
+
+👤 **Aldo Colombo**
+
+- GitHub: [@acolombo1](https://github.com/acolombo1)
+- Twitter: [@aldocolombo](https://twitter.com/aldocolombo)
+- LinkedIn: [Aldo Colombo](https://www.linkedin.com/in/aldo-colombo-2156009)
+## Show your support
+
+Give a ⭐️ if you like this project!
+## Acknowledgments
+
+- ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
[OPTIONAL] The `MIT link` and `live link` in the README file are broken. I would highly recommend fixing those sections to make README professional.